0001_initial.py

```diff
@@ -0,0 +1,141 @@
+# Generated by Django 2.2.1 on 2019-07-17 21:07
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django_extensions.db.fields
+
+
+class Migration(migrations.Migration):
```

Comment:
This migration file shouldn't be here. Everything else looks great. Wonderful job tackling this huge assignment!

Reply:
thanks..it's great to finally get this up and running...
that new file was introduced when i was trying to squash the commits..please check to make sure all other migration files are okay too...cause i had to resolve some conflicts from the migration files...